packages/web/src/js/utils/__tests__/Config.test.ts

```diff
@@ -174,6 +174,7 @@ describe('Config', () => {
       const obj = new DummyConfigClass();
       const config = {
         toggle: true,
+        // eslint-disable-next-line no-undefined
```

Comment:
Would you drop a comment with the reason why the rule is turned off?

Reply:
Yep, dropped.